app/models/test_track/fake/schema_loading.rb

```diff
@@ -0,0 +1,34 @@
+class TestTrack::Fake::SchemaLoading
```

Comment:
also, worth it to make this a singleton? otherwise if we have multiple uses it'll load the same schema file twice, right?